runtime/src/main/kotlin/kotlin/time/MonoClock.kt

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+package kotlin.time
+
+import kotlin.system.*
+
+public actual object MonoClock : LongReadingClock(unit = DurationUnit.NANOSECONDS), Clock { // TODO: interface should not be required here
```

Comment:
Indeed, API looks unnecessary object oriented.

Reply:
Interface should not be required, because it's already implemented, but still has to be mentioned to make MPP expect-actual matcher happy.

Reply:
I somewhat dislike using unneeded `object` on fast path (i.e. timestamps), where we pay init check every access. Could you please redesign it to be a top level function?

Reply:
`ClockMark` is now an abstract class instead of an interface, so it should perform better on repeated measuring.